sql/base/characters_database.sql

```diff
@@ -2510,7 +2510,7 @@ CREATE TABLE `updates` (
 
```

Comment:
The extra winner column should also be added to the create statement in here too.